ironmon_tracker/Tracker.lua

```diff
@@ -77,9 +77,13 @@ function Tracker.getPokemon(slotNumber, isOwn)
 			until not isEggPokemon or nextSlot == slotNumber
 		end
 		return Tracker.Data.ownPokemon[personality]
-	else
-		return Tracker.Data.otherPokemon[personality]
+	elseif Battle.isGhost then
+		-- Return Ghost dummy instead of showing the hidden mon's data, but keep the level
+		local retPokemon = Tracker.getGhostPokemon()
```

Comment:
Return Ghost pokemon data if fighting a ghost and getting enemy mon data

Reply:
I like this solution